dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/action/ImportContentletsAction.java

```diff
@@ -219,9 +220,9 @@ public void run() {
 							int languageCodeHeaderColumn = -1;
 							int countryCodeHeaderColumn = -1;
 							
-							byte[] bytes = (byte[]) httpReq.getSession().getAttribute("file_to_import");
+							byte[] bytes = (byte[]) httpSession.getAttribute("file_to_import");
```

Comment:
I am not 100% sure but I think using the request in a separated thread could be dangerous, depending on the load of the server the thread could be executed in a moment than the object is not longer available, could be better to take the values you need and pass to the thread, they could be defined as a final in the parent thread and them use in the child. 
Also, I am not sure about the context but should be this a thread, if so, shouldn't be executed by the DotConcurrent

Reply:
That's why I had to refactor this logic to avoid sending the http request. However, the session could not be removed because it implied changing the signature of public methods